packages/angular_devkit/build_optimizer/src/transforms/class-fold.ts

```diff
@@ -80,6 +127,19 @@ function findClassDeclarations(node: ts.Node): ClassData[] {
   const classes: ClassData[] = [];
   // Find all class declarations, build a ClassData for each.
   ts.forEachChild(node, (child) => {
+    // Check if it is a named class declaration first.
+    // Technically it doesn't need a name in TS if it's the default export, but when downleveled
+    // it will be have a name (e.g. `default_1`).
+    if (ts.isClassDeclaration(child) && child.name && ts.isIdentifier(child.name)) {
```

Comment:
`ts.isIdentifier` is unneeded here.  `name` is either undefined or an `Identifier`.

Reply:
Fixed